packages/sdk/src/sdkBase.ts

```diff
@@ -436,12 +432,14 @@ export class NxtpSdkBase {
       methodContext,
     );
 
-    const totalFee = gasFee.add(relayerFee).add(routerFee);
+    const totalGasFee = gasFee.add(relayerFee).add(gasFee);
+    const totalFee = totalGasFee.add(routerFee);
+    const receiverAmount = BigNumber.from(swapAmount).sub(totalGasFee);
```

Comment:
Ok, so the routerFee has already been applied to the returned `receivingAmount` (here called `swapAmount`), so we only need to apply the totalGasFee. Right?

And I guess `totalFee` is just used for logging below?